website/docs/api/cli.md

```diff
@@ -5,7 +5,7 @@ sidebar_position: 0
 # Command Line Interface
 
 The DeviceScript command line (CLI) allows to compile and debug programs from your favorite IDE.
-The CLI also also usable within containers (Docker, GitHub Codespaces, CodeSandbox, ...).
+The CLI also usable within containers (Docker, GitHub Codespaces, CodeSandbox, ...).
```

Comment:
doubled terms are occasionally intentional, but I don't think that's the case here